clusterloader2/testing/density/config.yaml

```diff
@@ -55,42 +69,55 @@ steps:
       labelSelector: kind = latencyPod
 - name: Creating saturation pods
 # Create latency pods
+- measurements:
```

Comment:
For consistency - the comment should be before the measurement.

Reply:
Done.